pkg/netpol/scan/scan.go

```diff
@@ -249,6 +266,15 @@ var singleResourceK8sKinds = map[string]struct{}{
 	CronJob:               {},
 	ReplicationController: {},
 	Service:               {},
+	Route:                 {},
+	Ingress:               {},
+}
+
+// the kinds that scan considers as networking policies objects
+var policyelements = map[string]struct{}{
+	Networkpolicy: {},
+	Route:         {},
+	Ingress:       {},
```

Comment:
I don't think Ingress and Route should be considered as policy elements, as I would still like to see a warning if the input dir for the analysis has no network policies. maybe this is another category of elements. consider how to rewrite the function `hasWorkloadsOrNetworkPolicies` as well (it assumes only netpol or workload resources ).